collisions/Readme.md

```diff
@@ -1,225 +1,71 @@
+- `ACCNB` is not a UID. It kind of serves as one starting in 2013, but prior to that the number would reset annually, and even now, the TPS-derived `ACCNB`s might repeat every decade. Derived tables use `collision_no`, as defined in `collisions_replicator.collision_no`. You could use `REC_ID` if you want an "involved persons" level ID.
+- `ACCNB` is generated from TPS and CRC counterparts when data is loaded into the Oracle database. It can only be 10 characters long (by antiquated convention). 
+  - TPS GO numbers, with format `GO-{YEAR_REPORTED}{NUMBER}`, (example: `GO-2021267847`) are converted into `ACCNB` by:
+      - extracting `{NUMBER}`, 
+      - zero-padding it to 9 digits (by adding zeros before the `{NUMBER}`), and 
+      - adding the last digit of the year as the first digit (and that's how `GO-2021267847` becomes `1000267847`). 
+  - CRC collision numbers are recycled annually, with convention dictating that the first CRC number be `8000000` (then the next `8000001`, etc.). To convert these to `ACCNB`s: 
+      - take the last two digits of the year and 
+      - add the 'year' digits as the first two digits of the `ACCNB` (so `8000285` reported in 2019 becomes `198000285`). 
+  - The length of each `ACCNB` is used to determine the source of the data for the `data_source` column in `collisions_replicator.events` (since TPS `ACCNB`s have 10 digits while CRC `ACCNB`s have nine).
+  - To keep the dataset current, particularly for fatal collisions, Data Collections will occasionally manually enter collisions using information directly obtained from TPS, or from public media. These entries may not follow `ACCNB` naming conventions. When formal data is transferred from TPS, they are manually merged with these human-generated entries.
+- The date the collision was reported is not included in `acc_safe_copy`.
+- Some rows are derived from other rows by the Data & Analytics team; for example `LOCCOORD` is a simplified version of `ACCLOC`.
+- Categorical data is coded using numbers. These numbers come from the Motor Vehicle Accident (MVA) reporting scheme.
+- Categorical data codes are stored in the `collision_factors` schema as tables. Each table name corresponds to the categorical column in `collisions_replicator.acc_safe_copy`. These are joined against `collisions_replicator.acc_safe_copy` to produce the derived materialized views.
+- Some columns, such as the TPS officer badge number, are not included due to privacy concerns. The most egregious of these are only available in the original Oracle database, and have already been removed in the MOVE server data.
+- TPS and CRC send collision records once they are reported and entered into their respective databases, which often leads to collisions being reported months, or even years, after they occurred. 
```

Comment:
I think this and the below points are more of a `limitation` or `caveat` or things to need to know than fun facts, as they are very important to understand :meow_determined:

Reply:
Implemented!